app/assets/javascripts/application.js

```diff
@@ -11,7 +11,8 @@
 // about supported directives.
 //
 //= require jquery
-//= require jquery-ui/autocomplete
+//= require jquery-ui
+//= require jquery-ui/widgets/autocomplete
```

Comment:
Apparently this is the new way of handling these dependencies https://github.com/jquery-ui-rails/jquery-ui-rails